services/data-info/src/data_info/routes/data.clj

```diff
@@ -5,15 +5,20 @@
         [data-info.routes.domain.stats])
   (:require [data-info.services.create :as create]
             [data-info.services.metadata :as meta]
+            [data-info.services.manifest :as manifest]
+            [clojure.tools.logging :as log]
             [data-info.services.entry :as entry]
             [data-info.services.write :as write]
             [data-info.services.page-file :as page-file]
             [data-info.services.page-tabular :as page-tabular]
             [data-info.util.config :as cfg]
+            [tree-urls-client.middleware :refer [wrap-tree-urls-base]]
             [clojure-commons.error-codes :as ce]
             [data-info.util.service :as svc]
             [data-info.util.schema :as s]))
 
+(defn tree-urls-middleware [handler] (wrap-tree-urls-base handler cfg/tree-urls-base-url))
```

Comment:
I have no idea why this didn't work otherwise, but I had to split this out into a function or compojure-api/clojure/whatever just spit out stupid errors at me. It also has to be public. Too much macro magic, presumably.